homeassistant/components/aladdin_connect/cover.py

```diff
@@ -28,25 +26,33 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up the Aladdin Connect platform."""
-    acc: AladdinConnectClient = hass.data[DOMAIN][config_entry.entry_id]
+    session: api.AsyncConfigEntryAuth = config_entry.runtime_data
+    acc = AladdinConnectClient(session)
```

Comment:
session is always used to create a `AladdinConnectClient`. Let's move this to `__init__.py` so we don't have to create this twice.